src/test/regress/expected/locally_execute_intermediate_results.out

```diff
@@ -893,21 +882,19 @@ NOTICE:  executing the command locally: WITH a AS (SELECT max(ref_table_1.key) A
 -- everything could be executed locally on the coordinator,
 -- but not on the worker
 WITH cte_1 as (SELECT * FROM table_1),
-cte_2 AS (SELECT * FROM cte_1),
-cte_3 AS (SELECT max(key) FROM cte_2)
+cte_2 AS MATERIALIZED (SELECT * FROM cte_1),
```

Comment:
I guess it'd be nicer if we can change `cte_1` to `MATERIALIZED` as well (here and in the following tests).